Documentation/build-container-examples/README.md

```diff
@@ -0,0 +1,71 @@
+# Build container examples
+
+This directory includes two examples of scripts that build ACI images.
+One is a generic [PostgreSQL][postgresql] image and the other is an example [Play Framework][play-framework] app that connects to a PostgreSQL database.
+The example Play Framework app expects a PostgreSQL database, so both images can be used in a single pod to make the example run.
+
+## PostgreSQL
+
+The PostgreSQL image is based on [Alpine Linux][alpine-linux] and defines two mounts, one for a PostgreSQL data directory that will be used if present, and another for an SQL script that can be used to initialize the database.
+By default, if no data is found in the PostgreSQL data directory, a database named `rkt` will be created with user and password set to `rkt` too.
+Then, the initialization SQL script will run.
+If a PostgreSQL data directory is found in the corresponding mount, it will be used instead.
+
+## Example Play Framework app
+
+This is a very simple [example Play Framework][play-framework-example] app that just prints the number of times a page is retrieved.
+This value is stored in a PostgreSQL database.
+
+## Build the images
+
+You can build the images by running their build scripts.
+Assuming a privileged shell is running in this directory.
+
+```
+# (cd postgres && ./build-postgres.sh)
```

Comment:
```
# (cd postgres && ./build-postgres.sh)
./build-postgres.sh: line 12: acbuild: command not found
```

Could you either quickly say how to install `acbuild` in this documentation, or get the script to print it in an error message?

A link to https://github.com/containers/build/releases could be enough.

Reply:
Done.